src/main/java/frc/robot/commands/DriveManual.java

```diff
@@ -12,23 +15,27 @@
 import frc.robot.Constants.constDrivetrain;
 import frc.robot.Constants.constField;
 import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.StateMachine.DriverState;
 
 public class DriveManual extends Command {
   Drivetrain subDrivetrain;
   DoubleSupplier xAxis, yAxis, rotationAxis;
   boolean isOpenLoop;
   double redAllianceMultiplier = 1;
+  StateMachine subStateMachine;
 
   public DriveManual(Drivetrain subDrivetrain, DoubleSupplier xAxis, DoubleSupplier yAxis,
       DoubleSupplier rotationAxis) {
```

Comment:
Stat machine must be parameter. Remember to check your code with a simulate or check if the PR build failed